src/components/templates/PartyMap/components/RoomModal/RoomModal.scss

```
@@ -14,6 +14,14 @@ $spacing: 20px;
     background-size: cover;
   }
 
+  &__room-info {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    padding: $spacing;
```

Top padding just adds unnecessary space and makes text uneven with image.

```suggestion
    padding: 0 $spacing $spacing;
```